mobify/platform-scaffold
# WEB- 1498: Add to homescreen more menu

This PR adds the more menu in Add to homescreen mode. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1498
 **Linked PRs**: depends on https://github.com/mobify/platform-scaffold/pull/952

## Changes
- Add more menu in add to homescreen mode
- Add the share component to the more menu

## Todos

## How to test-drive this PR
- checkout the `WEB-1498__A2HS-more-menu` branch of the SDK
- cd into the SDK folder
- run `npm link`
- cd into the scaffold folder
- run `npm link progressive-web-sdk`
- run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- browse to https://www.merlinspotions.com/?homescreen=1
- click the more icon in the top right
- links inside the menu should work correctly
- the menu should not be visible if you remove
